deepchecks/nlp/text_data.py

```diff
@@ -87,17 +87,22 @@ def __init__(
     ):
 
         # Require explicitly setting task type if label is provided
-        if task_type is None:
+        if task_type in [None, 'other']:
             if label is not None:
-                raise DeepchecksValueError('task_type must be set when label is provided')
-            self._task_type = None
+                if isinstance(label, t.Sequence):
+                    if pd.notnull(label).any():
+                        raise DeepchecksValueError('task_type must be set when label is provided')
+                else:
+                    raise DeepchecksValueError('task_type must be set when label is provided')
+
+            self._task_type = TaskType.OTHER
```

Comment:
Did you add OTHER to the enum? Can't find it here

Reply:
It was already there, I didn't need to add